admin/class-admin-apple-index-page.php

```diff
@@ -405,6 +407,25 @@ private function push_action( $id ) {
 		}
 	}
 
+	/**
+	 * Handles a debug action.
+	 *
+	 * @param int|null $id The ID of the post being debugged.
+	 *
+	 * @access private
+	 */
+	private function debug_action( ?int $id ): void {
+		$post = get_post( $id );
+		if ( ! $post ) {
+			return;
+		}
+
+		$action = new Apple_Actions\Index\Push( $this->settings, $id );
```

Comment:
does this actually push it? Is that what we want to happen?

Reply:
No, just calls the construct and send in the settings, we only call the debug method, not the push method, the debug info is print to the screen and then exits. I'll add a bit of documentation to the README.

Reply:
Added some docs here: https://github.com/alleyinteractive/apple-news/wiki/Usage#debug-api-requests